src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/Win32Program.cs

```diff
@@ -622,7 +630,7 @@ private static ParallelQuery<Win32> PathEnvironmentPrograms(string[] suffixes)
                         .ToArray();
 
             var programs1 = allPaths.AsParallel().Where(p => Extension(p).Equals(ShortcutExtension, StringComparison.OrdinalIgnoreCase)).Select(LnkProgram);
-            var programs2 = allPaths.AsParallel().Where(p => Extension(p).Equals(ApplicationReferenceExtension, StringComparison.OrdinalIgnoreCase)).Select(Win32Program);
+            var programs2 = allPaths.AsParallel().Where(p => Extension(p).Equals(ApplicationReferenceExtension, StringComparison.OrdinalIgnoreCase)).Select(Create);
```

Comment:
I'm just wondering if `Create` is the best name that we can have for the function which was previously `Win32Program` because all the other functions which do a similar function are named as shortcut followed by Program.

Reply:
Yeah I'm open to suggestions. Win32 conflicts with the Win32 namespace so i renamed it win32 program, but then that was redundant with the Win32 this 'builder function'.  I've used 'Create' in the past when the using static 'builder' methods.  I've also used 'GetInstance' but generally reserve that for singletons.   Alternatives might be CreateWin32Program or BuildWin32Program?

Reply:
I think CreateWin32Program sounds good. It gives us information about the app type as well. Thanks!

Reply:
Sounds good, I'll update.  Thanks 👍